### PR TITLE
Remove the heart icon button to add the view as a favorite from the top bar

### LINKED ISSUE #8546: Remove the heart icon button to add the view as a favorite from the top bar
Image

Add in ⌘K instead.

### PR DESCRIPTION
**Pull request by ehconitin**
closes #8546 

@Bonapara please check the behaviour, if this is what you were looking for! ;)

### CODE REVIEW

File: packages/twenty-front/src/modules/views/view-picker/components/ViewPickerOptionDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          closeDropdown();
+        }}
+        dropdownId={`view-picker-options-${view.id}`}
+        dropdownContent={
+          <DropdownMenuItemsContainer>
+            {isIndexView ? (
+              isFavoriteFolderEnabled && (
+                <MenuItem
+                  LeftIcon={IconHeart}
+                  text={'Favorite picker'}
```
Comment (ehconitin) [inline packages/twenty-front/src/modules/views/view-picker/components/ViewPickerOptionDropdown.tsx:78]: @Bonapara I was not sure what to call this :), for now I have named the menu item `Favorite picker`

**Comment by Bonapara** [comment]
Hey @ehconitin! 

I cannot see the add/remove from favorites button in the view switcher 🤔

https://github.com/user-attachments/assets/f2b7166d-8f94-4b81-82f0-7ee5eba21f29

**Comment by ehconitin** [comment]
@Bonapara 
The feature flag must be disabled on your end. Could you make sure IsFavoriteFolderEnabled and IsFavoriteFolderEntityEnabled is enabled :) Thanks!

**Comment by Bonapara** [comment]
@ehconitin we would like the view to be added directly to the favorite root level when we press "Add to Favorites". When a view is already part of favorites, the CTA should be called "Manage Favorite".

Also, could you display the "Favorite" menu to the right of the `3 dots menu` instead of on the left?

**Comment by martmull** [comment]
Functional comments:
- wrong left margin before the heart Icon
image
- dropdown should be displayed above the other onces, not replacing them
image

**Review by martmull — APPROVED** [review]

**Pull request merged**